### PR TITLE
vimc-6929 update ow api url

### PR DESCRIPTION
**Pull request by hillalex**
Updates OW and the task queue.

I also committed the gradle and compiler settings for Intellij, since opening up in a fresh install defaults to the worng version of java without these.

### CODE REVIEW

File: scripts/start-task-queue.sh [hunk 1]
```diff
@@ -37,7 +38,7 @@ docker run -d \
 
 # add task q user
 CLI=vimc/montagu-cli:master
-docker pull $CLI
+#docker pull $CLI
```
Comment (EmmaLRussell) [inline scripts/start-task-queue.sh:41]: Should this be commented out?

File: src/config/blackboxTests/config.properties [hunk 2]
```diff
@@ -1 +1 @@
-orderlyweb.api.url=http://orderly_web_web:8888/api/v1
+orderlyweb.api.url=http://orderly_web_web:8888/api/v2
```
Comment (EmmaLRussell) [inline src/config/blackboxTests/config.properties:1]: I guess this is going to need updating in montagu settings too?
https://github.com/vimc/montagu/tree/master/settings

Comment (hillalex) [reply src/config/blackboxTests/config.properties:1]: yes PR here https://github.com/vimc/montagu/pull/228

File: scripts/start-task-queue.sh (hunk 1 above)
Comment (hillalex) [reply scripts/start-task-queue.sh:41]: no, my bad! will edit

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**